frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import { styled, Typography } from '@mui/material';
+
+import { formatAssetPath } from '../../../utils/formatPath';
+import { FC } from 'react';
+
+export const StyledFigure = styled('figure')(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(2),
+    flexDirection: 'column',
+}));
+
+export const StyledFigCaption = styled('figcaption')(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(2),
+    flexDirection: 'column',
+}));
+export const StyledTitle = styled(Typography)({
```

Comment:
Based on the usage (and my understanding from the screenies you posted), it feels like this should be an `h2` element if I'm reading it right? But that's a minor thing, and I'll leave it up to you, whether or not to handle that.

```suggestion
export const StyledTitle = styled('h2')({
```